docs/cli.md

```diff
@@ -155,6 +155,24 @@ mainflux-cli channels get <channel_id> <user_auth_token>
 mainflux-cli things connect <thing_id> <channel_id> <user_auth_token>
 ```
 
+#### Bulk Connect Things to Channels
+
+```bash
+mainflux-cli provision connect <file> <user_auth_token>
+```
+
+* `file` - A CSV or JSON file containing thing and channel ids
+* `user_auth_token` - A valid user auth token for the current system
+
+An example file might be
+
+```csv
+7f3216a2-f5c8-4410-a37e-a1bbd7f48231,f5dc42c8-0bf1-4850-bb0e-ddea435e7814
+f26688e7-8bdf-4ec8-b1dc-1bfb764cfa3a,ca3365f9-2202-40a6-a9d6-75b6690df466
```

Comment:
Sorry, but could you use `<thing_id>` and `<channel_id>`? And also add an example for JSON, please?